subprojects/core/src/main/java/org/gradle/api/tasks/ant/AntTarget.java

```diff
@@ -33,6 +35,16 @@ public abstract class AntTarget extends ConventionTask {
     private Target target;
     private File baseDir;
 
+    public AntTarget() {
+        getDegradationController().requireConfigurationCacheDegradation(this,
+            getProject().getProviders().provider(() -> "Task is not compatible with the Configuration Cache")
```

Comment:
💅 

We usually do not capitalize "the configuration cache" in messages

Reply:
Should I change this? I see that you also used capitalized configuration cache. Either option is fine with me